### DISCUSSION TITLE
Github Sponser asks to create a stripe account but stripe doesn't allow international payment for individual users

### DISCUSSION THREAD

**Discussion by dotexe3301** [community/community#147615] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I am from India and wanted to enable GitHub Sponsor on my account. Github sponsors need a stripe account, so I head over to create one. But, stripe doesn't allow individuals to accept international payments you should be a company or sole proprietor or something like that. 

So, how can I enable GitHub sponsor on my account with this problem?

**Comment by mecodeatlas** [comment]
Hey @dotexe3301! Thanks for reaching To use GitHub Sponsors, you need to create a **Stripe Connect account** directly through the GitHub dashboard instead of a regular Stripe account. Did you follow this flow within GitHub to set up your account?  

If you're trying to set up a US-based account, keep in mind that proof of US residency is required. Without it, this setup won’t be possible. If you need further guidance, please, open a ticket on our [Support page](https://support.github.com/contact) :)

Best of luck!

**Reply by dotexe3301** [reply, marked as answer]
I created it from GitHub. But It didn't work!

**Reply by digital-carver** [reply]
I'm also facing the same issue. I was following the Github Sponsors UI/wizard, and it redirected me to a connect.stripe.com page. And in that page it says "Accepting international payments is not available for individual user."

![image](https://github.com/user-attachments/assets/aedbca73-29d8-455b-9c4d-ef7b8195ebc3)

**Reply by crizmo** [reply]
Any updates on this ?

**Reply by fatalcoder524** [reply]
Any Updates?

**Reply by Cprakhar** [reply]
+1
